pkg/skaffold/event/v2/status_check.go

```diff
@@ -18,11 +18,14 @@ package v2
 
 import (
 	"fmt"
+	"strconv"
 
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/constants"
 	proto "github.com/GoogleContainerTools/skaffold/proto/v2"
 )
 
+var resourceIDs = map[string]int{}
```

Comment:
is there a way to avoid this static map? I added a `ID()` method to `resource.Deployment` which I think can be used here.

Reply:
I think I can just use the resource name as the id, since it shouldn't make a difference on the ide side and that's what the integer ids would map to anyway